pkg/volume/util/fs/fs_windows.go

```diff
@@ -58,7 +59,8 @@ func FsInfo(path string) (int64, int64, int64, int64, int64, int64, error) {
 
 // DiskUsage gets disk usage of specified path.
 func DiskUsage(path string) (*resource.Quantity, error) {
-	_, _, usage, _, _, _, err := FsInfo(path)
```

Comment:
It looks like the only other usage of `FsInfo` is in https://github.com/kubernetes/kubernetes/blob/5d6e4829a881fdc8862554698abbd3ae00c50d0f/pkg/volume/metrics_statfs.go#L57-L69. Is that subject to the same bug?